qadence_protocols/mitigations/twirl.py

```diff
@@ -67,23 +60,53 @@ def mitigate(
     samples_twirl_den_list = []
     for twirl in twirls:
         layer = [X(i) for i in twirl]
-        block_twirl = chain(circuit.block, kron(*layer))
+        block_twirl = chain(block, kron(*layer))
```

Comment:
```suggestion
        block_twirl = chain(block, kron(X(i) for i in twirl))
```

Please try it but I am almost certain it works.

Reply:
Yup it works